data/channels.csv

```diff
@@ -35037,6 +35043,7 @@ WorldPokerTour.fr,World Poker Tour,WPT,Pluto TV,,FR,,,c/FR,fra,sports,FALSE,,,,h
 WorldPokerTour.it,World Poker Tour,,Pluto TV,,IT,,,c/IT,ita,sports,FALSE,,,,https://pluto.tv/en/live-tv/world-poker-tour-it,https://i.imgur.com/e3yJ9QN.png
 WorldPokerTour.uk,World Poker Tour,,Pluto TV,,UK,,,c/UK,eng,sports,FALSE,,,,https://pluto.tv/en/live-tv/world-poker-tour-gb,https://i.imgur.com/znqxz2W.png
 WorldPokerTour.us,World Poker Tour,,,,US,,,r/INT,eng,sports,FALSE,,,,https://www.worldpokertour.tv/watch/channel/live/live/hFz9XetVSqRy-pluto,https://i.imgur.com/98kLMjj.png
+WorldTurk.tr,World Türk TV,,,,TR,,,c/TR,tur,,FALSE,,,,https://worldturk.com.tr/world-turk-tv-canli-yayin-izle/,https://www.artidijitalmedya.com/image/400x400/cdn/channels/v1/logo_151.png

```

Comment:
![image](https://github.com/user-attachments/assets/415f6ab6-2f1f-43fe-8245-f4d3bc025231)
tvg-id must be WorldTurkTV.tr
channel name must be World Turk TV
alternate name must be World Türk TV